### PR TITLE
revert: bring back gtm

### LINKED ISSUE #419: Google Tag Manager is not triggered in the new homepage
## Description

Previous Google Tag Manager is no longer triggered in Warga Bantu Warga website after the homepage revamps, thus creating an issue where data is not sent to Google Analytics. 

## Expected Behaviour

Google Tag Manager should be triggered on every page, see the previous PR here: https://github.com/kawalcovid19/wargabantuwarga.com/pull/36

## Additional Information

- [x] Add the GTM trigger back to WBW
- [x] Add proper noscript implementation for Google Tag Manager in WBW
- [ ] Add unit test to this critical functionality to ensure that future refactor will be less likely to produce the same bug, or later engineers will notice if this bug is recurring. Perhaps using React Test Library or Enzyme #425

See the discussion thread here: Screen Shot 2021-07-25 at 22 31 49

### PR DESCRIPTION
**Pull request by mazipan**
Closes #419

## Description

Based on this PR #320, I revert the changes about GTM, move it to the top level

## Current Tasks

- [x] Take the code from #320
- [x] Bring back to the top level, not the page level
- [x] Remove double GTM on the `home-page.tsx`

### CODE REVIEW

**Comment by baraeb92** [comment]
Is this something that we can add unit test to? 

Just to make sure similar issue less likely happening in the future? @mazipan

**Comment by mazipan** [comment]
I think we can cover this case by our automated testing.
Will create on seperate PR.

Since this is a high priority bug, we need to deliver the fix as fast as we can.

**Comment by mazipan** [comment]
Create #425 for the testing part

**Review by resir014 — APPROVED** [review]

File: pages/_app.tsx [hunk 1]
```diff
@@ -82,6 +83,25 @@ export default function App({ Component, pageProps, router }: AppProps) {
         <link href="/manifest.json" rel="manifest" />
         <meta content="#1667C2" name="theme-color" />
       </Head>
+
```
Comment (baraeb92) [inline pages/_app.tsx:86]: This line 86 could be removed I think

File: pages/_app.tsx [hunk 2]
```diff
@@ -82,6 +83,25 @@ export default function App({ Component, pageProps, router }: AppProps) {
...
+        }}
+      />
+      <noscript>
+        <iframe
+          height="0"
+          src="https://www.googletagmanager.com/ns.html?id=GTM-5X4ZPBX"
+          style={{ display: "none", visibility: "hidden" }}
+          title="gtm"
+          width="0"
+        />
```
Comment (baraeb92) [inline pages/_app.tsx:103]: need to add </iframe> instead for the noscript to work properly. 

See previous implementation here: https://github.com/kawalcovid19/wargabantuwarga.com/pull/36/files?file-filters%5B%5D=.tsx

**Review by baraeb92 — COMMENTED** [review]
Just some concern about this mas @mazipan @zainfathoni 

Need to make sure that the noscript is also present on the body, currently, when loading the web, it seems that there's no noscript present when I check the source code.

File: pages/_app.tsx (hunk 2 above)
Comment (zainfathoni) [reply pages/_app.tsx:103]: in React, they should be identical, Mas @baraeb92.

But somehow it's not showing in the page now, could you please investigate it further, Mas @mazipan?

Comment (mazipan) [reply pages/_app.tsx:103]: Bisa insert expectation kode komplit nya dari GTM gak ya, Mas @baraeb92.?

Comment (mazipan) [reply pages/_app.tsx:103]: Since all this code are just `copy-paste` from previous implementation without any modifications

Comment (mazipan) [reply pages/_app.tsx:103]: Since all this code are just copy-paste from previous implementation without any modifications

Comment (mazipan) [reply pages/_app.tsx:103]: Screen Shot 2021-07-26 at 10 50 41

It already exist, it just under `main` tag, should we move it under `body`?

**Pull request merged**